<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DNSEditor</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>771</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>默认适配器：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dns_adapter">
        <property name="text">
         <string>XXXXXX</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>首要DNS:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dns_main">
        <property name="text">
         <string>114.114.114.114</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>副DNS:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dns_second">
        <property name="text">
         <string>114.114.114.114</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="dns_widget">
      <property name="batchSize">
       <number>93</number>
      </property>
      <property name="selectionRectVisible">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="text">
         <string>添加DNS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_del">
        <property name="text">
         <string>删除DNS</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_main">
          <property name="text">
           <string>设置为首要DNS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_second">
          <property name="text">
           <string>设置为副DNS</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_back">
        <property name="text">
         <string>恢复初始状态</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
